extensions/src/AWSSDK.Extensions.Bedrock.MEAI/JsonContext.cs

```diff
@@ -0,0 +1,77 @@
+﻿/*
```

Comment:
Since the class is called `BedrockJsonContext`, should we rename the file here too?